_jobs/junior-java-developer.html

```diff
@@ -0,0 +1,113 @@
+image: /img/jobs/spring-boot.png
+---
+<div class="job">
+    <section id="one" class="wrapper spotlight style1">
+        <div class="inner">
+            <div class="content">
+                <h2 class="major">Who are we looking for?</h2>
+                <p>
+                    We strive to remain thought leaders in the ever changing technology market.
+                    JWorks makes this a personal goal of each of our colleagues to stay on top of the game. 
+                    This can be done by following or contributing to numerous <span class="accentuate">WORKSHOPS</span>, coding sessions and webinars we organise, 
+                    but what’s more fun than going to <span class="accentuate">INTERNATIONAL CONFERENCES</span>
+                    such as Spring I/O in <span class="accentuate">BARCELONA</span> or MongoDB World in <span class="accentuate">NEW YORK</span>? 
+                    Take a couple of days extra to enjoy the surroundings and get to know your colleagues better!
+                </p>
+                <ul>
+                    <li>Software <span class="accentuate">development</span> is your true passion</li>
+                    <li>You contribute in the successful delivery of our project</li>
+                    <li>You make sure that our clients are proud on your contribution</li>
+                    <li><span class="accentuate">Solving complex issues</span> doesn't scare you</li>
+                    <li>You are interested in the <span class="accentuate">latest trends</span> and evolutions and you want to learn them yourself</li>
+                    <li>You collaborate in our Competence Centers and you share your knowledge with your colleagues</li>
+                </ul>
+            </div>
+        </div>
+    </section>
+
+    <section id="two" class="wrapper alt spotlight style2">
+        <div class="inner">
+            <div class="content">
+                <h2 class="major">What are the requirements?</h2>
+                <ul>
+                    <li>Bachelor or Master level, or equal by experience</li>
+                    <li>Multilingual Dutch and / or French and English</li>
+                    <li>You work customer oriented and have a service mindset</li>
+                    <li>You are a team player</li>
+                    <li>You take initiative and you are very eager to learn</li>
+                    <li>You are flexible and stress resistant</li>
+                </ul>
+            </div>
+        </div>
+    </section>
+
+
+    <section id="three" class="wrapper spotlight style3">
+        <div class="inner">
+            <div class="content">
+                <h2 class="major">Technology stack</h2>
+                <ul>
+                    <li>Spring stack, including Spring Boot, Spring MVC, Spring Data and Spring Cloud</li>
+                    <li>Automated testing (unit testing, integration testing and acceptance testing) of applications doesn't have any secrets for you</li>
```

Comment:
I would also mention something about knowing how to work with a mocking framework.